### PR TITLE
publiccloud : replace typing of real text with env variables

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: lib/publiccloud/provider.pm [hunk 1]
```diff
@@ -189,8 +189,8 @@ sub run_img_proof {
     $cmd .= '--no-cleanup ';
     $cmd .= '--collect-vm-info ';
     $cmd .= '--service-account-file "' . $args{credentials_file} . '" ' if ($args{credentials_file});
-    $cmd .= "--access-key-id '" . $args{key_id} . "' " if ($args{key_id});
-    $cmd .= "--secret-access-key '" . $args{key_secret} . "' " if ($args{key_secret});
+    #TODO: this if is just dirty hack which needs to be replaced with something more saine ASAP.
+    $cmd .= "--access-key-id \$AWS_ACCESS_KEY_ID --secret-access-key \$AWS_SECRET_ACCESS_KEY" if(is_ec2());
```
Comment (mimi1vx) [inline lib/publiccloud/provider.pm:193]: small 
`$cmd .= '--access-key-id $AWS_ACCESS_KEY_ID --secret-access-key $AWS_SECRET_ACCESS_KEY' if (is_ec2)`

Comment (mimi1vx) [reply lib/publiccloud/provider.pm:193]: + hated perltidy ..

**Review by mimi1vx — APPROVED** [review]

File: lib/publiccloud/provider.pm (hunk 1 above)
Comment (asmorodskyi) [reply lib/publiccloud/provider.pm:193]: ok I will try that

File: lib/publiccloud/provider.pm [hunk 2]
```diff
@@ -189,8 +189,8 @@ sub run_img_proof {
     $cmd .= '--no-cleanup ';
     $cmd .= '--collect-vm-info ';
     $cmd .= '--service-account-file "' . $args{credentials_file} . '" ' if ($args{credentials_file});
-    $cmd .= "--access-key-id '" . $args{key_id} . "' " if ($args{key_id});
-    $cmd .= "--secret-access-key '" . $args{key_secret} . "' " if ($args{key_secret});
+    #TODO: this if is just dirty hack which needs to be replaced with something more saine ASAP.
```
Comment (grisu48) [inline lib/publiccloud/provider.pm:192]:
```suggestion
    #TODO: this if is just dirty hack which needs to be replaced with something more sane ASAP.
```

Comment (asmorodskyi) [reply lib/publiccloud/provider.pm]: fixed

**Pull request merged**